### PR TITLE
Fix typos in the OSSL_METHOD_STORE doc

### PR DESCRIPTION
**Pull request by newsuk-jzelenka**
Fixed some typos that I noticed when reading OSSL_METHOD_STORE docs.

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/internal/man3/OSSL_METHOD_STORE.pod [hunk 1]
```diff
@@ -88,24 +88,24 @@ and the ones passed to the ossl_method_store_free().
...
 ossl_method_store_cache_get() queries the cache associated with the B<store>
-for an method identified by B<nid> that matches the property query
+for a method identified by B<nid> that matches the property query
 B<prop_query>.
 The result, if any, is returned in B<method>.
 
 ossl_method_store_cache_set() sets a cache entry identified by B<nid> with the
 property query B<prop_query> in the B<store>.
-Future cache gets will return the specified B<method>.
+Future calls to ossl_method_store_cache_get will return the specified B<method>.
```
Comment (InfoHunter) [inline doc/internal/man3/OSSL_METHOD_STORE.pod]: I suggest to change `ossl_method_store_cache_get` to `ossl_method_store_cache_get()`

Comment (levitte) [reply doc/internal/man3/OSSL_METHOD_STORE.pod]: Good point

**Comment by newsuk-jzelenka** [comment]
Closing as it's from wrong account.

File: doc/internal/man3/OSSL_METHOD_STORE.pod (hunk 1 above)
Comment (newsuk-jzelenka) [reply doc/internal/man3/OSSL_METHOD_STORE.pod]: Ok will address it in the new PR created from the right account.

Comment (bukka) [reply doc/internal/man3/OSSL_METHOD_STORE.pod]: Fixed in #9806 that's also created under right account.

**Pull request closed without merging**